backend/src/routes/user.ts

```diff
@@ -3,10 +3,15 @@ import { Request, Response, Router } from 'express';
 import { Types } from 'mongoose';
 import { ExerciseEvent } from '../models/exercise-event';
 import { IUser, User } from '../models/user';
+import { upload } from '../s3';
 import { handleErrors } from '../util';
 
 const saltLength = 11;
 
+interface IMulterFile {
+    location: string;
+}
+
```

Comment:
you shouldn't create this interface, you should import it from @types/multer-s3.

Reply:
done